utils/build.config

```diff
@@ -7,7 +7,7 @@ PMDK = 1.11.0
 ISAL = v2.30.0
 ISAL_CRYPTO = v2.23.0
 SPDK = v21.07
-OFI = v1.14.0rc3
+OFI = v1.14.0
```

Comment:
once ofi changes landed patch on line 19 has to be updated to include new patch as well